frontend/src/components/Home/Home.css

```diff
@@ -1,10 +1,467 @@
+@import url('http://fonts.cdnfonts.com/css/abeezee');
+
+*{
+    font-family: 'ABeeZee', sans-serif;
+}
+
+/* NAVBAR COMPONENT */
 .color-nav{
     background-color: #8FC0A9;
 }
 
+/* BUTTONS */
+.button-link{
+    text-align: center;
+}
+
+.button-link button, .button-link a{
```

Comment:
I believe this will apply changes to all 'button' and all  'a'  divisions on the page, is that what we are trying to achieve or just a division with the classname: button-link

Reply:
same comment on span, svg, h1, and input below

Reply:
The space between them is a descendant connector, which means that it will only affect the selectors inside the class.